routes/cartRouter.js

```diff
@@ -3,6 +3,10 @@ import { cartController } from '../controllers';
 
 const router = express.Router();
 
-router.get('/', cartController.getCategory);
+router.post('/:id', cartController.addCartList);
+router.get('/', cartController.getCartList);
```

Comment:
POST ->`http://localhost:8000/carts/1` 

```suggestion
router.post('', cartController.addCartList);
router.get('/', cartController.getCartList);
```
`productId`라는 데이터는 request.body 에 있어야 합니다